presidio-analyzer/analyzer/entity_recognizer.py

```diff
@@ -38,9 +38,13 @@ def __init__(self, supported_entities, name=None, supported_language="en",
 
         loglevel = os.environ.get("LOG_LEVEL", "INFO")
         self.logger = logging.getLogger(__name__)
+        logging.basicConfig(
+            format='%(asctime)s:%(levelname)s:[%(req_id)s]%(message)s',
+            level=loglevel)
+
         self.logger.setLevel(loglevel)
         self.load()
-        logging.info("Loaded recognizer: %s", self.name)
+        logging.info("Loaded recognizer: %s", self.name, extra={'req_id': ''})
```

Comment:
Is this still required?

Reply:
I was not aware of this line, good catch. I'll remove it